### ISSUE TITLE
KeyError: 'completed_orders' in get_order_history()

### ISSUE THREAD

**Issue by arjuneshm** [krishnavelu/alice_blue#369]
Hi,
Krishna

I am frequently checking the order history of my orderId . Consider it is less than second .

Following is the error i am getting, looks like the order history dictionary output is missing in between when i am querying. 
Traceback (most recent call last):

    orders = alice.get_order_history()["data"]["completed_orders"]

KeyError: 'completed_orders'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):

Basically my script almost ran for 15 minutes in the loop where it was able to check the status continuously.
If you are able to judge, please guide me with some solution.

Thanks and  Regards
Arjunesh

**Comment by arjuneshm** [comment]
On further debug i understood that , after some 5 to 10minutes, there is a break happening for the output too. But that is not creating the exception.


Unfortunately this is random (unvertain - but certain in a day - atleast 2 times) , so a guess would help me to tackle this issue.

Code used for checking : 

while(True):
                orders=alice.get_order_history()["data"]["completed_orders"]
                print(f'orders is {orders}')

**Comment by arjuneshm** [comment]
Till now the maximum time the continous query of orderhistory ran was around 1 hr 15mins .

**Comment by krishnavelu** [comment]
This is a python issue, learn python.

**Comment by arjuneshm** [comment]
Hi,
Krishna

I didn't really get that because the command is giving output for more than an hour and sometimes its not giving data and giving an error. 
As of now i am handling this with try exception with retry seeing some of the sample codes.

Thanks and Regards
Arjunesh

**Comment by krishnavelu** [comment]
not always the response will contain ["data"]["completed_orders"]. So you need to handle that.
Only if present, try to extract ["data"]["completed_orders"]

**Comment by krishnavelu** [comment]
Hope you problem is resolved. Closing this.
